### DISCUSSION TITLE
Form手动验证后通过后怎么取值？

### DISCUSSION THREAD

**Discussion by langziyang** [Tencent/tdesign-react#2891] [Q&A]
form.validate()
            .then(resp => {
                console.log(resp)
//postToApi
            })

为什么resp要返回一个true呢？验证通过后不是应该返回数据吗？
只返回一个true，那我在这里怎么取值呢？只能onSubmit?

**Comment by HaixingOoO** [comment]
返回`true`是验证通过，然后可以使用`form?.getFieldsValue(true)`获取所有值，失败的话会返回`失败的校验结果列表`，不过你有更好的建议可以说一下

**Reply by langziyang** [reply]
倒不一定我说的就正确，既然是 Promise的话验证不通过的数据是否应该放在error里用reject里返回更好呢？then里只需要验证通过的数据

**Reply by HaixingOoO** [reply]
嗯，后续可以优化一下(可能会放在v2版本吧)，但是目前改动的话会影响很大

**Reply by cccnoob** [reply]
请问 如何单独校验 FormList 中的某一个字段 比如这样 form.validate({fields: [’address.0.area‘, ’address.1.area‘]}); 校验前两行的 area 字段？
